static/app/views/releases/thresholdsList/thresholdGroupRows.spec.tsx

```diff
@@ -34,35 +33,21 @@ describe('ThresholdGroupRows', () => {
     ...thresholdData,
   });
 
-  const wrapper = (org: Organization = organization) => {
-    return function WrappedComponent({children}) {
-      return (
-        <OrganizationContext.Provider value={org}>
-          {children}
-        </OrganizationContext.Provider>
-      );
-    };
-  };
-
-  type RenderProps = ThresholdGroupRowsProps & {org: Organization};
-  const DEFAULT_PROPS: RenderProps = {
+  const DEFAULT_PROPS: ThresholdGroupRowsProps = {
     allEnvironmentNames: ['test'],
     project: ProjectFixture(),
     refetch: () => {},
     setTempError: () => {},
-    org: organization,
```

Comment:
just moving org to the 2nd argument since its not actually a prop on ThresholdGroupRows but is being passed in as DEFAULT_PROPS